### PR TITLE
feat: My-profile view and update details

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/KiboPasswordForm.vue [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    <SfInput
+      v-for="(field, key) in fields"
+      :key="key"
+      :name="field.id"
+      :label="field.label"
+      :value="field.value"
+      class="form__element"
+      type="password"
+      :valid="isConfirmPasswordSame === null || isConfirmPasswordSame"
+      :error-message="field.id === 'confirmPassword' ? `Password doesn't match` : ''"
```
Comment (kibo-sushant) [inline components/KiboPasswordForm.vue]: use localization

File: composables/useUser.ts [hunk 2]
```diff
@@ -154,6 +155,47 @@ export const useUser = () => {
...
+      },
+    }
+    try {
+      loading.value = true
+      await fetcher({
+        query: updatePasswordMutation,
+        variables,
+      })
+    } catch (err) {
+    } finally {
```
Comment (kibo-sushant) [inline composables/useUser.ts:198]: console the error

File: components/KiboPasswordForm.vue [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      <div class="password-requirements__header">{{ $t("Password Requirements") }}</div>
+      <SfCharacteristic
+        v-for="(req, key) in requirements"
+        :key="key"
+        color-icon="black"
+        size-icon="md"
+        class="password-requirements__details"
+      >
+        <template #icon>
+          <SfIcon size="1.25rem">
```
Comment (kibo-sushant) [inline components/KiboPasswordForm.vue:25]: I think size should be always px (pass it without px)

File: composables/useUser.ts [hunk 4]
```diff
@@ -154,6 +155,47 @@ export const useUser = () => {
...
+        variables,
+      })
+      user.value = response.data?.updateCustomerAccount
+    } catch (err) {
+    } finally {
+      loading.value = false
+    }
+  }
+
+  const changePassword = async ({ oldPassword, newPassword }) => {
```
Comment (kibo-sushant) [inline composables/useUser.ts:179]: Don't see test cases for covering updateCustomerPersonalData  and changePassword methods. Have we written it?

Comment (kibo-chandradeeptalaha) [reply composables/useUser.ts:179]: Done

**Review by kibo-kevinwatts — APPROVED** [review]

**Review by kibo-sushant — APPROVED** [review]

**Pull request merged**